server/app/services/ti/TrustedIntermediaryService.java

```diff
@@ -111,6 +118,143 @@ private Optional<String> validateDateOfBirth(String dob) {
     return Optional.empty();
   }
 
+  private Form<EditTiClientInfoForm> validateFirstNameForEditClient(
```

Comment:
In the next work Item, Melanie will be re-using the same view for both Edit and Add Clients. So the form will also be reused (If I understand it correctly.) Then it will be easier to re-use these validation methods.

Reply:
Gotcha, that makes sense.